### PR TITLE
UCT/IB/MLX5: Don't fail w/o DEVX KSM support

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8776/ for details (Mellanox internal link).

**Comment by Artemy-Mellanox** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8777/ for details (Mellanox internal link).

File: src/uct/ib/mlx5/ib_mlx5_dv.c [hunk 1]
```diff
@@ -99,8 +99,8 @@ static ucs_status_t uct_ib_mlx5dv_create_ksm(uct_ib_md_t *ibmd,
     memh->atomic_dvmr = mlx5dv_devx_obj_create(md->super.dev.ibv_context, in, inlen,
                                                out, sizeof(out));
     if (memh->atomic_dvmr == NULL) {
-        ucs_error("CREATE_MKEY KSM failed: %m");
-        status = UCS_ERR_IO_ERROR;
+        ucs_debug("CREATE_MKEY KSM failed: %m");
```
Comment (yosefe) [inline src/uct/ib/mlx5/ib_mlx5_dv.c]: do we need to use UMR, or atomics will just have bad perf?

Comment (Artemy-Mellanox) [reply src/uct/ib/mlx5/ib_mlx5_dv.c]: bad perf, UMR blocked the same way

**Pull request merged**